app/views/dradis/plugins/calculators/dread/_ce_tools_menu.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if !defined?(Dradis::Pro) %>
+  <li>
+    <%= link_to 'Risk Calculators - DREAD', dread_calculator.calculators_dread_path, class: 'dropdown-item', data: { turbo: false } %>
+  </li>
+<% end %>
```

Comment:
We don't need this guard since Pro will never be defined in CE, and this will never be called from Pro.
```suggestion
<li>
  <%= link_to 'Risk Calculators - DREAD', dread_calculator.calculators_dread_path, class: 'dropdown-item', data: { turbo: false } %>
</li>
```